WordPressAuthenticator/Unified Auth/View Related/Login/MagicLinkRequestedViewController.swift

```diff
@@ -0,0 +1,158 @@
+import UIKit
+import WordPressUI
+
+final class MagicLinkRequestedViewController: LoginViewController {
+
+    // MARK: Properties
+
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var subtitleLabel: UILabel!
+    @IBOutlet private weak var emailLabel: UILabel!
+    @IBOutlet private weak var cannotFindEmailLabel: UILabel!
+    @IBOutlet weak var buttonContainerView: UIView!
+    @IBOutlet weak var loginWithPasswordButton: UIButton!
```

Comment:
Nit: We could make `buttonContainerView` and `loginWithPasswordButton` private as well.